### ISSUE TITLE
Pengwin Needs Better Documentation

### ISSUE THREAD

**Issue by sirredbeard** [WhitewaterFoundry/Pengwin#493]

**Comment by sirredbeard** [comment]
Coming to https://pengwin.readthedocs.io/en/latest/

**Comment by swebs** [comment]
It would be great to know what the `pengwin-setup` options actually do for example.

**Comment by sirredbeard** [comment]
You can go look at the code in the pengwin-setup repository.

**Comment by swebs** [comment]
Yep, I figured that out shortly after posting that it was easy... for anyone reading, just go into the `pengwin-setup.d` directory and read the obviously named scripts there.

https://github.com/WhitewaterFoundry/pengwin-setup/tree/master/pengwin-setup.d
